charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl

```diff
@@ -120,3 +120,20 @@ rules:
 {{- end }}
 {{- end -}}
 {{- end -}}
+
+{{- define "kube-apiserver.watchCacheSizes" -}}
+{{- with .Values.watchCacheSizes }}
```

Comment:
Wouldn't be easier to read and understand with usage of `if statements` for emptiness instead of `with`. Currently `with` sets the value of `watchCacheSizes` to dot (`.`) and I was wondering from where comes `.resources` and `.default`. WDYT?

```suggestion
{{- if .Values.watchCacheSizes }}
# do something
{{- end }}
{{- if .Values.watchCacheSizes.resources }}
# do something
{{- end }}
```

Reply:
I did it like you suggested before and I wasn't happy with the long value references so I switched to `with` to make it more readable. But no strong opinion :)